tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -340,7 +340,7 @@ public function data_provider_to_test_various_strategy_dependency_chains() {
 <script id="async-with-blocking-dependency-js-before" type="text/javascript">
 scriptEventLog.push( "async-with-blocking-dependency: before inline" )
 </script>
-<script type='text/javascript' src='https://example.com/external.js?script_event_log=async-with-blocking-dependency:%20script' id='async-with-blocking-dependency-js' async></script>
+<script type='text/javascript' src='https://example.com/external.js?script_event_log=async-with-blocking-dependency:%20script' id='async-with-blocking-dependency-js' async  data-wp-strategy='async'></script>
```

Comment:
I see an extra space appearing here:

```suggestion
<script type='text/javascript' src='https://example.com/external.js?script_event_log=async-with-blocking-dependency:%20script' id='async-with-blocking-dependency-js' async data-wp-strategy='async'></script>
```

Reply:
Fixed thanks @westonruter , I am surprised this did not fail.

Reply:
It didn't fail because the whitespace wasn't significant to cause a difference in the parsed DOM tree, now that we're using `assertEqualMarkup`:

https://github.com/10up/wordpress-develop/blob/575c8373ce45f911bc00c9ccad6afa61656e5ed6/tests/phpunit/tests/dependencies/scripts.php#L717

This eliminates differences being raised by varying quote styles, whitespace, etc.